zkevm-circuits/src/util/cell_manager.rs

```diff
@@ -122,6 +129,27 @@ impl<F: Field> Expr<F> for &Cell<F> {
     }
 }
 
+#[derive(Clone)]
+pub(crate) struct CellValueOnly {
```

Comment:
It seems that `CellValueOnly` 
- is pretty specific to keccak logic however defined in shared place
- it's just a subset of Cell<F> with quite of field being remove.

Can we just follow the design on old `Cell<F>` https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/keccak_circuit/cell_manager.rs#L35-L43 and extend existing `Cell<F>` with a new `new_value_only` api so we do not implemenet dedicated api for `CellValueOnly`

Reply:
The thing with this is that Column would need to be an Option, do you think that is the best way?

I am seeing that actually all fields in Cell could be private, so we could simplify the code, while putting column as Option. I am going to try

Reply:
I think `Option<Column>` make sense 👍  since we have this use case in keccak